File: GamesTournamentsWeb.Web/ClientApp/models/tournaments/Prize.ts
import { ConvertableToJson } from '~/models/ConvertableToJson'

class Prize extends ConvertableToJson {
  place: number
  amount: number
  currency: string

  constructor (place: number, amount: number, currency: string) {
    super()

    this.place = place
    this.amount = amount
    this.currency = currency
  }
}

export { Prize }

File: GamesTournamentsWeb.DataAccess/Models/Tournaments/Prize.cs
namespace GamesTournamentsWeb.DataAccess.Models.Tournaments;

public class Prize
{
    public int Id { get; set; }
    public int Place { get; set; }
    public int Amount { get; set; }
    public int CurrencyId { get; set; }
    public Currency Currency { get; set; }
    public int TournamentId { get; set; }
    public Tournament Tournament { get; set; }
}